src/main/kotlin/net/ccbluex/liquidbounce/utils/render/TargetRenderer.kt

```diff
@@ -68,15 +78,147 @@ class WorldTargetRenderer(module: ClientModule) : TargetRenderer<WorldRenderEnvi
     val legacy = Legacy()
     val circle = Circle(module)
     val glowingCircle = GlowingCircle(module)
+    val ghost = Ghost()
 
     override val appearance =
         choices<Choice>(
             module,
             "Mode",
-            { glowingCircle },
-            { arrayOf(legacy, circle, glowingCircle) }
+            { ghost },
```

Comment:
Default should stay on glowing circle, thanks.

Reply:
okay